api/Document.json

```diff
@@ -10626,17 +10626,6 @@
                     "value_to_set": "true"
```

Comment:
Missing `version_removed` in this entry, but I can't comment on the appropriate line.